test/ring/middleware/test/anti_forgery.clj

```diff
@@ -68,6 +68,21 @@
       (is (= (get-in response [:session ::af/anti-forgery-token])
              (:body response))))))
 
+(deftest token-in-request-test
+ (letfn [(handler
+           ([request] {:status 200 :headers {} :body (:anti-forgery-token request)})
+           ([request response _] (response (handler request))))]
```

Comment:
You've put in `response` here, rather than `respond`. Can you change it so that it uses the standard naming scheme? (One-arity handlers use `request`, while three-arity use `request`, `respond` and `raise`.)